developer-docs-site/docs/guides/system-integrators-guide.md

```diff
@@ -358,6 +373,10 @@ Each event in `events` is differentiated by an `key`. The `key` is derived from
 
 Using the above example, `events[1].key` is `0x02000000000000005098df8e7969b58ab3bd2d440c6203f64c60a1fd5c08b9d4abe6ae4216246c3e`. This is equivalent to `changes[0].data.data.deposit_events.guid`, which is `{"addr": "0x5098df8e7969b58ab3bd2d440c6203f64c60a1fd5c08b9d4abe6ae4216246c3e", "creation_num": "2"}`.
 
+:::tip
```

Comment:
line 357 to 359 above (Each event in `events`...) needs to be cleaned up. Perhaps we can just get rid of this nonsense and say how guid => guid?